R/vars.R

```diff
@@ -14,10 +14,17 @@ peeker <- function(what) {
         fn <- glue::glue("`{fn}()`")
       }
 
+      if (is_interactive() && cli::ansi_has_hyperlink_support()) {
```

Comment:
Should you check for `cli::num_ansi_colors() != 1` as well to fix the snapshots?

Technically we should also use the feature guard but this will eventually be handled by cli.

Reply:
Maybe to get the tests passing it's better to set the other env var too